Sources/EduIDExpansion/Flows/Home/MainCoordinator.swift

```diff
@@ -86,5 +89,13 @@ extension MainCoordinator: OnBoardingMainNavigationDelegate {
         homeNavigationController.presentedViewController?.dismiss(animated: true)
         children.removeAll { $0 is OnboardingCoordinator }
     }
+}
 
+extension MainCoordinator: ActivitMainNavigationDelegate {
+    
+    //MARK: - activity flow methods
+    func dismissActivityFlow(sender: AnyObject) {
+        homeNavigationController.presentedViewController?.dismiss(animated: true)
+        children.removeAll { $0 is ActivityCoordinator }
```

Comment:
The 'sender' here is the child coordinator to remove. Hence why you'd normally define this delegate method something like `func coordinatorDidFinish(coordinator: MyCoordinator)`.

Reply:
I resolved this by removeAll { $0 === sender }

Reply:
Are there still unpushed changes?